fefix/src/fixs/mod.rs

```diff
@@ -79,6 +79,8 @@ impl Version {
                     .unwrap();
                 context.set_options(SslOptions::NO_COMPRESSION);
                 context.set_options(SslOptions::NO_SESSION_RESUMPTION_ON_RENEGOTIATION);
+                // NO_TLSV1_3 requires openssl version 1.1.1
+                #[cfg(openssl111)]
```

Comment:
`openssl111` needs to be available to crate users via `Cargo.toml` or detected and set automatically during build time by `build.rs`. As of now, this attribute has no effect whatsoever.